api/app/Models/PoolCandidate.php

```diff
@@ -234,41 +236,59 @@ public function filterByPools(Builder $query, array $pools): Builder
         return $query;
     }
 
-    public function scopeHasDiploma(Builder $query, bool $hasDiploma): Builder
+    public function filterByEquity(Builder $query, array $equity): Builder
     {
-        if ($hasDiploma) {
-            $query->where('has_diploma', true);
+        if (empty($equity)) {
+            return $query;
         }
-        return $query;
-    }
-    public function scopeHasDisability(Builder $query, bool $hasDisability): Builder
-    {
-        if ($hasDisability) {
-            $query->where('has_disability', true);
+
+        // OR filter - first find out how many booleans are true, create array of all true equity bools
+        // equity object has 4 keys with associated bools
+        $equityVars = [];
+        if ($equity["is_woman"]) {
+            array_push($equityVars, "is_woman");
         }
-        return $query;
-    }
-    public function scopeIsIndigenous(Builder $query, bool $isIndigenous): Builder
-    {
-        if ($isIndigenous) {
-            $query->where('is_indigenous', true);
+        if ($equity["has_disability"]) {
+            array_push($equityVars, "has_disability");
         }
-        return $query;
-    }
-    public function scopeIsVisibleMinority(Builder $query, bool $isVisibleMinority): Builder
-    {
-        if ($isVisibleMinority) {
-            $query->where('is_visible_minority', true);
+        if ($equity["is_indigenous"]) {
+            array_push($equityVars, "is_indigenous");
+        }
+        if ($equity["is_visible_minority"]) {
+            array_push($equityVars, "is_visible_minority");
+        }
+
+        // then return queries depending on above array count, special query syntax needed for case of >=2 to ensure proper SQL query formed
+        if (count($equityVars) == 0) {
```

Comment:
Are these branches for count == 0 and count == 1 necessary?  I think it would work without them?

Reply:
I'll leave in the handle 0 for a just in case, but yeah, == 1 doesnt seem needed